routers/repo/http.go

```diff
@@ -331,6 +351,37 @@ func HTTP(ctx *context.Context) {
 	ctx.NotFound("Smart Git HTTP", nil)
 }
 
+func pushCreateRepo(authUser, owner *models.User, repoName string) (*models.Repository, error) {
+	if !authUser.IsAdmin {
+		if owner.IsOrganization() {
+			team, err := owner.GetOwnerTeam()
+			if err != nil {
+				return nil, err
+			}
+			if !team.IsMember(authUser.ID) {
```

Comment:
I think there is already a new option to allow other teams to create repository. That PR sent after this PR and merged before this one.

Reply:
Done